pyanaconda/modules/storage/storage.py

```diff
@@ -113,21 +113,11 @@ def __init__(self):
 
         # Initialize the partitioning modules.
         # TODO: Remove the static partitioning modules.
-        self._auto_part_module = self.create_partitioning(PartitioningMethod.AUTOMATIC)
-        self._add_module(self._auto_part_module)
-
-        self._manual_part_module = self.create_partitioning(PartitioningMethod.MANUAL)
-        self._add_module(self._manual_part_module)
-
-        self._custom_part_module = self.create_partitioning(PartitioningMethod.CUSTOM)
-        self._add_module(self._custom_part_module)
-
-        self._interactive_part_module = self.create_partitioning(PartitioningMethod.INTERACTIVE)
-        self._add_module(self._interactive_part_module)
-
-        self._blivet_part_module = self.create_partitioning(PartitioningMethod.BLIVET)
-        self._add_module(self._blivet_part_module)
-
+        self._add_module(self.create_partitioning(PartitioningMethod.AUTOMATIC))
```

Comment:
Shouldn't you remove also the `TODO` note?

Reply:
No, because the static modules should be removed completely.